edb/server/main.py

```diff
@@ -286,9 +294,9 @@ def run_server(args: ServerConfig):
                 abort('Could not start database cluster in %s',
                       args.data_dir)
 
-            need_cluster_restart = _init_cluster(cluster, args)
-
-            if need_cluster_restart and pg_cluster_started_by_us:
+            need_cluster_restart = _init_cluster(cluster, args,
+                                                 pg_cluster_init_by_us)
```

Comment:
I'm not sure this is correct.  `pg_cluster_init_by_us` indicates whether or not we ran `initdb`.  It would always be `False` for RDS instances and so `--init-command` wouldn't get executed.  Instead, let's move everything, including the creation of the default database under the `if new_template_db_id` and stop doing "partial bootstrap".

Reply:
Hm, yes, now I see that `_ensure_edgedb_template_database(..)` is actually our flag. But I'm still not sure why we need those alter database: https://github.com/edgedb/edgedb/blob/ea3a6ec9196a5c0f6bcd34f80a31bf562eff369a/edb/server/bootstrap.py#L207-L230
It looks like we want to skip any database mutations if DB is bootstrapped (i.e. when template database exists). Including the cited snippet and creating superuser at: https://github.com/edgedb/edgedb/blob/ea3a6ec9196a5c0f6bcd34f80a31bf562eff369a/edb/server/bootstrap.py#L1004-L1011 
Or am I misunderstanding something?

Reply:
Yes, skip all mutations post-bootstrap (i.e. move them into the bootstrap process).

Reply:
Fixed.